modules/apps/27-interchain-accounts/types/expected_keepers.go

```diff
@@ -31,6 +31,7 @@ type ChannelKeeper interface {
 	SendPacket(ctx sdk.Context, channelCap *capabilitytypes.Capability, packet ibcexported.PacketI) error
 	ChanCloseInit(ctx sdk.Context, portID, channelID string, chanCap *capabilitytypes.Capability) error
 	ChanOpenInit(ctx sdk.Context, order channeltypes.Order, connectionHops []string, portID string, portCap *capabilitytypes.Capability, counterparty channeltypes.Counterparty, version string) (string, *capabilitytypes.Capability, error)
+	CounterpartyHops(ctx sdk.Context, channel channeltypes.Channel) ([]string, bool)
```

Comment:
All middleware will need to implement these functions (including fees) cc @AdityaSripal

Reply:
These are included in #446 to review also! For example, `ChanOpenInit` and `ChanCloseInit` above are unused afaik

Reply:
We should remove unused interface functions

Reply:
I can take care of this in a smaller follow up PR 👍